packages/ui-options/src/Options/index.tsx

```diff
@@ -104,11 +104,12 @@ class Options extends Component<OptionsProps> {
 
   renderLabel() {
     const { renderLabel, styles } = this.props
+    const isAndroidOrIOS = /Android|iPhone|iPad/i.test(navigator.userAgent)
     return (
       <span
         id={this._labelId}
         role="presentation"
-        aria-hidden="true"
+        aria-hidden={isAndroidOrIOS ? 'false' : 'true'}
```

Comment:
Can you please leave a comment detailing why this is needed?

Reply:
Rewrote the code with ua-parser-js and left a comment too